### DISCUSSION TITLE
getRouteKey() for email generation

### DISCUSSION THREAD

**Discussion by dominikgeimer** [spatie/laravel-welcome-notification#35] [Q&A]
Hello,

when generating the email the user id is in the url (of course), but I would like to use a Hashid instead of the user id. 

Is it possible to use the getRouteKey for URL generation?

```
public function getRouteKey()
    {
        return Hashids::encode($this->getKey());
    }
```

**Comment by freekmurze** [comment]
I guess this is possible, and if not, feel free to PR it 👍

**Reply by dominikgeimer** [reply]
Thanks for your reply. I figured it out:

```
public function welcomeNotificationKeyValue()
    {
        return $this->getRouteKey();
    }
```
